.circleci/config.yml

```diff
@@ -20,3 +34,7 @@ workflows:
   check-standards:
     jobs:
       - standards
+      - bundle:
+          filters:
```

Comment:
What are y'alls thoughts on how frequently this build should run? Should we limit it to specific branches?

Reply:
@bmo-tv Thanks for this! Filtering on tags seems right, the same as [AB Pro](https://github.com/studiopress/atomic-blocks-pro/blob/46effb3b9e7be1f9fdd7b22562e78f70ae0aedcf/.circleci/config.yml#L140).

Only tagging of the master branch should trigger a zip and deploy if you wanted to restrict by branch, although we do not tend to tag other branches.

Reply:
Updated the filters to isolate this job to only the `master` branch

Reply:
@bmo-tv Thanks! Looks good.